libs/platform/table/components/table-toolbar/table-toolbar.component.ts

```diff
@@ -230,6 +230,16 @@ export class TableToolbarComponent implements TableToolbarInterface {
         this._table.expandAll();
     }
 
+    /** @hidden */
+    _formatAppliedFilters(appliedFilters: { columnName: string; params: string }[]): string {
+        return appliedFilters
+            .map((filter, index) => {
+                const separator = index < appliedFilters.length - 1 ? ', ' : '';
+                return `${filter.columnName} (${filter.params})${separator}`;
```

Comment:
It looks like this is working well to me but I am curious if this is the behavior we would be looking for with RTL languages. Not sure how that works